Formula/up.rb

```diff
@@ -17,28 +17,28 @@
 class Up < Formula
   desc 'The official Upbound CLI'
   homepage 'https://upbound.io'
-  version 'v0.14.0'
+  version 'v0.15.0'
   license 'Upbound Software License'
 
   if OS.mac? && Hardware::CPU.intel?
-    url 'https://cli.upbound.io/stable/v0.14.0/bundle/up/darwin_amd64.tar.gz'
-    sha256 '628d0a74d129a5832de089027cb7ab8bd28c7ad5a08f5f2b73cdf5205ed490f2'
+    url 'https://cli.upbound.io/stable/v0.15.0/bundle/up/darwin_amd64.tar.gz'
+    sha256 '84ccce60cd8a819ac5eaf6d0548c1625c167f5797bd941bb2736a9a06a66bba3'
   end
   if OS.mac? && Hardware::CPU.arm?
-    url 'https://cli.upbound.io/stable/v0.14.0/bundle/up/darwin_arm64.tar.gz'
-    sha256 '5fad8d569244b219a84340a245fe52a0e0ce3e47e574c9b1ea0e016ccd91e006'
+    url 'https://cli.upbound.io/stable/v0.15.0/bundle/up/darwin_arm64.tar.gz'
+    sha256 '98be20f2b94db2e49baa74d86e6ca3381a5af534677cf6bb49f76891c536f998'
   end
   if OS.linux? && Hardware::CPU.intel?
-    url 'https://cli.upbound.io/stable/v0.14.0/bundle/up/linux_amd64.tar.gz'
+    url 'https://cli.upbound.io/stable/v0.15.0/bundle/up/linux_amd64.tar.gz'
     sha256 'fa9817753a131f6b9f8c3be77c0c6f5b35e01925052b68ea32e64c88f0399db2'
```

Comment:
this seems wrong 🤔

Reply:
Looks like we had an issue here where old version was propagated -- I have fixed manually for the time-being.